src/components/gds-menu-item-nested/gds-menu-item-nested.scss

```diff
@@ -2,136 +2,108 @@
 
 :host {
   display: block;
+  position: relative;
 }
 
-.active {
-  @include mobileOnly {
-    background-color: var(--menu-item-background-color-active, var(--background-color-primary));
+@include mobileOnly {
```

Comment:
Should this be also tablet?

Reply:
No idea, I just used what was there before but I'll confirm it! Good point